packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -18,7 +18,7 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
   /**
    * Callback issues when the selected option changes
    */
-  onChanged?: (option: IDropdownOption, index?: number) => void;

```

Comment:
I think we were operating under the assumption that combo box was still a beta component, but I am now seeing it on the documentation site, so we probably can't make a breaking change here. What about deprecating in master and removing in 5.0 branch?